src/robot/api/deco.py

```diff
@@ -68,3 +68,43 @@ def decorator(func):
         func.robot_types = types
         return func
     return decorator
+
+
+def library(scope=None, version=None, robot_auto_kws=False):
```

Comment:
1. `robot_auto_kws` is inconsistent compared to other arguments. `auto_keywords` would be better.
2. `auto_keywords` should be `True` by default! There should be a test that it can be disabled, though.
3. We also have two other attributes libraries can have: `ROBOT_LIBRARY_DOC_FORMAT` and `ROBOT_LIBRARY_LISTENER`. Should add matching `doc_format` and `listener` arguments here as well.
4. Hard to say what would be the best order. Perhaps this: `scope, version, doc_format,  listener, auto_keywords`